task-5/results/monitoring/grafana/provisioning/datasources/prometheus.yml

```
@@ -0,0 +1,26 @@
+  - name: Elasticsearch
+    type: elasticsearch
+    access: proxy
+    url: http://elasticsearch-batch:9200
+    database: "spring-batch-*"
+    jsonData:
+      interval: Daily
+      timeField: "@timestamp"
+      esVersion: "8.8.0"
+      logMessageField: message
+      logLevelField: level
+    editable: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)